src/plugins/biostruct3d_view/src/deprecated/BioStruct3DGLWidget.cpp

```diff
@@ -79,6 +79,19 @@ void BioStruct3DGLWidget::tryGL() {
     Q_UNUSED(wgt);
 }
 
+bool BioStruct3DGLWidget::canRender() {
+    QGLWidget w;
```

Comment:
Have you tested current and deprecated variants of the method?

Reply:
> Are there any details why UGENE does not work in your environment correctly?

I don't know why it is not drawn. On Windows with OpenGL 1.1, [this example](https://doc.qt.io/qt-5/qtgui-openglwindow-example.html) works. However, 3D models are not rendered in UGENE (for both deprecated and non-deprecated code). On another computer with Windows and OpenGL 3.x, 3D models are rendered

> Have you tested current and deprecated variants of the method?

Yes

Reply:
If you have a Windows installation that can run QT OpenGL examples with no issues, but UGENE's version of OpenGL visualization fails it is becomes very important to find the cause of the problem and provide a fix.

In this patch you try to use very basic features of OpenGL and it fails. Looks like the problem is with OpenGL libraries lookup, build mode flags, etc..

I believe that by comparing the working & non-working build and runtime environments you will be able to find the delta that can make UGENE work and create a real fix for the issue.

Reply:
> >Have you tested current and deprecated variants of the method?
> 
> Yes

But locally in Windows. Don't know how non-deprecated version works on Linux

Reply:
I hope the UTI-259 will be done and, perhaps, will solve this problem too. I see no point in dealing with outdated code if the UTI-259 will possibly be done